sql-plugin/src/main/scala/com/nvidia/spark/rapids/AvroDataReader.scala

```diff
@@ -98,18 +144,16 @@ class AvroDataFileReader(si: SeekableInput) extends AutoCloseable {
   private var firstBlockStart: Long = 0
 
   // store all blocks info
-  private val blocks: ArrayBuffer[BlockInfo] = ArrayBuffer.empty
+  private val blocks: mutable.ArrayBuffer[BlockInfo] = mutable.ArrayBuffer.empty
```

Comment:
Nit: This is only setup during construction and therefore does not need to be mutable.